docs/contributors/code/react-native/getting-started-react-native.md

```diff
@@ -33,23 +33,23 @@ npm ci
 ## Run
 
 ```sh
-npm run native start:reset
+npm run other:native start:reset
```

Comment:
I think we should leave `native` commands unchanged. All those commands are related to the mobile apps developed with React Native. That's why it's perfectly fine they keep them grouped this way.

Reply:
Fixed with 3fb7df1 👍